concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -313,6 +316,22 @@ public Semver getVersion() {
         return new Semver(versionString);
     }
 
+    /**
+     * Get the list of smart contract instances in a given block.
```

Comment:
This is not entirely correct. 
The list returned will contain all instances that exists at the end of the given block. 

So something like "Get a list of all smart contract instances that exist in the state at the end of the given block." will better describe it.